src/garage/torch/misc/tensor_utils.py

```diff
@@ -0,0 +1,15 @@
+"""utility functions for PyTorch tensors."""
```

Comment:
i know this is the location for TF but we are trying to actually get rid of generic misc/utils modules.

Can you put this into a module inside torch.algos instead? You can call it `loss_function_utils.py`

Reply:
done